### PR TITLE
tried to fix issue#75

### PR DESCRIPTION
**Pull request by farhaanbukhsh**
Tried to fix it according to the requirements given

### CODE REVIEW

File: nuancier/templates/contribute_index.html [hunk 1]
```diff
@@ -170,7 +170,31 @@
...
+                <img src="{{ url_for('static', filename='Approved.png') }}"
+                    alt="Submission open"/>
+                {% else %}
+                <img src="{{ url_for('static', filename='Denied.png') }}"
+                    alt="Submission closed"/>
+                {% endif %}
+                {{ election.submission_date_start }}
+            </td>
+        </tr>
+        {% endfor %}
```
Comment (ralphbean) [inline nuancier/templates/contribute_index.html]: I think the placement of this ``{% endfor %}`` might be problematic.

If there are 5 elections, then this code will open 5 ``<table>`` tags, but only close once at the end of the loop.

Do you see what I mean @farhaanbukhsh?

Perhaps the ``{% endfor %}`` should be moved down a few lines?

**Comment by pypingou** [comment]
So this is how it looks today:
nuancier_contribute_date2
And this, how it looks with your changes:
nuancier_contribute_date

My thoughts:
* We loose the attractiveness of the big blue button
* It includes the start date of the submission but not the end date (which was the original idea behing the ticket #75)
* I'm not entirely set on the table if something else looks better

**Comment by farhaanbukhsh** [comment]
Seems legit so what should we do now? Retain the button or go with the table or something else?

**Comment by pypingou** [comment]
I think I'd keep the button and just add more information behind it

**Comment by farhaanbukhsh** [comment]
Even I like the button so instead of table i will add button and below button there will be submission date.
Your thoughts?

**Comment by pypingou** [comment]
Below or next to, either should work, feel free to share a screenshot :)

**Comment by farhaanbukhsh** [comment]
selection_019


This looks cool I didn't like putting date below the buttons , It gets confusing and looks shabby too.

**Comment by pypingou** [comment]
What is this date? The start of the submission? If so we know the submission is opened, otherwise the election wouldn't be listed on this page :)

What we need is *until* when are the submissions opened

**Comment by farhaanbukhsh** [comment]
selection_020

So we need to put votes open date here right? Do tell me the way it looks now.

**Comment by farhaanbukhsh** [comment]
selection_021

Is it okay now?

**Comment by pypingou** [comment]
I would use a different label, we're speaking about the submission phase here so why mentioning the vote?

What about using: `` submission ends on ...`` ? But then it would likely be better after the button rather than before.

**Comment by farhaanbukhsh** [comment]
Aye Aye Captain

**Comment by farhaanbukhsh** [comment]
selection_022

How about this?

**Comment by pypingou** [comment]
I think I liked it better when they were in the same line and I think we can drop the icon

**Comment by farhaanbukhsh** [comment]
selection_023

This looks okay

**Comment by farhaanbukhsh** [comment]
selection_024

now?

**Comment by pypingou** [comment]
Yeah, I like that :)

File: nuancier/templates/contribute_index.html [hunk 2]
```diff
@@ -169,9 +169,12 @@
         <a class="large_button" href="{{ url_for('contribute', election_id=election.id) }}">
             {{ election.election_name }} - {{ election.election_year }}
         </a>
+        <span>Submission ends on : </span>
+        {{ election.election_date_start }}
     </li>
     {% endfor %}
 </ul>
+</table>
```
Comment (pypingou) [inline nuancier/templates/contribute_index.html]: That one can be dropped

**Comment by pypingou** [comment]
Perfect thanks for this and your patience! :)

**Pull request merged**